packages/ag-charts-enterprise/src/series/bullet/bulletDefaults.ts

```diff
@@ -9,7 +9,6 @@ export const BULLET_DEFAULTS = {
             position: POSITION.LEFT,
             nice: false,
             max: undefined as number | undefined,
```

Comment:
Should these move too?

Reply:
It's the only place I wasn't sure about. WDYT?